src/annotator/anchoring/test/types-test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var types = require('../types');
+
+var TextQuoteAnchor = types.TextQuoteAnchor;
+var TextPositionAnchor = types.TextPositionAnchor;
+
+// These are primarily basic API tests for the anchoring classes. Tests for
+// anchoring a variety of HTML and PDF content exist in `html-test` and
+// `pdf-test`.
+describe('Anchoring classes', function () {
```

Comment:
That seems like a pretty reasonable convention. Perhaps files should be named after their default exports too